x-pack/plugins/cross_cluster_replication/public/__jest__/client_integration/auto_follow_pattern_list.test.js

```diff
@@ -269,7 +269,7 @@ describe('<AutoFollowPatternList />', () => {
       test('should have a "settings" section', () => {
         actions.clickAutoFollowPatternAt(0);
         expect(find('settingsSection').find('h3').text()).toEqual('Settings');
-        expect(exists('settingsValues')).toBe(true);
+        expect(exists('settingsValues', 4)).toBe(true);
```

Comment:
nit: I think we could we make this a bit more readable. Perhaps if we do a find on settings value and check against its lenght, not sure what you think about that!

```
// These are the different sections that reflect the options of an autofollow pattern
const AVAILABLE_SETTINGS = 4;
expect(find('settingsValues').length).toBe(AVAILABLE_SETTINGS);
```

Reply:
Thanks for the suggestion @sabarasaba! It looks good to me so I added it in the next commit.